src/node_api.h

```diff
@@ -116,21 +117,18 @@ NAPI_EXTERN napi_status napi_create_array_with_length(napi_env env,
                                                       size_t length,
                                                       napi_value* result);
 NAPI_EXTERN napi_status napi_create_number(napi_env env,
-                                           double val,
+                                           double value,
                                            napi_value* result);
 NAPI_EXTERN napi_status napi_create_string_utf8(napi_env env,
-                                                const char* s,
-                                                size_t length,
+                                                const char* buf,
+                                                size_t bufsize,
```

Comment:
I'm not sure I like the `buf`, `bufsize` parameter names here. I guess the intention was to make them consistent with the parameters in `napi_get_value_string_` APIs? But to me, "buf" implies a buffer that is *to be filled* by the function that is called.